llama_stack/providers/inline/post_training/huggingface/recipes/finetune_single_device.py

```diff
@@ -224,8 +225,8 @@ async def load_dataset(
             tuple: (train_dataset, eval_dataset, tokenizer)
         """
         # Validate data config
-        if not config.data_config:
-            raise ValueError("DataConfig is required for training")
+        if config.data_config is None:
```

Comment:
Why do we need this change as part of the standardization of the config error? Please address this separately, thanks!